.github/workflows/ci-windows-artifacts.yml

```diff
@@ -56,14 +56,15 @@ jobs:
 
       - name: Prepare libavif (cmake)
         run: >
-          cmake -G Ninja -S . -B build
+          cmake -G "Visual Studio 17 2022" -A x64 -S . -B build
           -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=OFF
           -DAVIF_CODEC_AOM=LOCAL -DAVIF_CODEC_AOM_ENCODE=ON
           -DAVIF_CODEC_AOM_DECODE=OFF -DAVIF_CODEC_DAV1D=LOCAL
           -DAVIF_LIBYUV=LOCAL -DAVIF_LIBSHARPYUV=LOCAL
           -DAVIF_JPEG=LOCAL -DAVIF_ZLIBPNG=LOCAL
           -DAVIF_BUILD_EXAMPLES=OFF -DAVIF_BUILD_APPS=ON
           -DAVIF_BUILD_TESTS=OFF -DAVIF_ENABLE_WERROR=ON
+          -DCMAKE_VS_WINDOWS_TARGET_PLATFORM_VERSION=10
       - name: Build libavif (ninja)
         working-directory: ./build
         run: ninja
```

Comment:
Now this will need an update to `cmake --build` or `msbuild`, no?

Reply:
Yes, but I started doing that in another PR https://github.com/AOMediaCodec/libavif/pull/2665, which may be too much of a change for no tangible benefit? Let me know if you have an opinion on that.

Reply:
> Yes, but I started doing that in another PR #2665, which may be too much of a change for no tangible benefit? Let me know if you have an opinion on that.

I'm OK with doing that assuming the performance of the presubmits was similar.